api/server.go

```diff
@@ -103,6 +109,11 @@ func NewServer(config common.Config) error {
 		jobsCache:  jobsCache,
 	}
 
+	d := scheduler{
```

Comment:
this is kinda nitpicky but why is the scheduler called `d`?

Reply:
because `s` was taken by `server` 😆

Reply:
ah, the limitation of single letter variables 😆

Reply:
🎖